src/Symfony/Component/Yaml/Dumper.php

```diff
@@ -30,6 +30,10 @@ class Dumper
      */
     public function __construct($indentation = 4)
     {
+        if ($indentation < 1) {
+            throw new \InvalidArgumentException('The indentation must be greater than zero.');
```

Comment:
The identation can be a float number like 3.14? or should ensure that the indentation is a natural number?